.tool-versions

```diff
@@ -0,0 +1,5 @@
+nodejs 18.18.2
+dotnet 3.1.426
+java temurin-11.0.21+9
+python 3.12.0
+golang 1.21.3
```

Comment:
same asdf versions as in GH actions

Reply:
we only use typescript. Why do we want to add java and others?

Reply:
this is if you want to test packaging: https://github.com/mongodb/awscdk-resources-mongodbatlas/blob/main/.github/workflows/package.yml#L53-L180

Reply:
packaging is run in the github action. I am not sure why we need the asdf tool for them since they are not used while developing on the CDK

Reply:
> but for instance in the doc it says to do a release from local if the release gh action fails, if that case you need to package.

I am not sure about this since the npx uses public (https://github.com/cdklabs/publib) and jsii (https://github.com/aws/jsii) which should not require running any of those tools for the release. Did you run the release manually? Did you have to install them? 

> I think this doesn't do any harm, and you are not forced to install all of those if you don't want to even having asdf installed

IMO I think that the harm is that it is confusing since the repo is in typescripts and they are not needed for the release (see above)

Reply:
ok, removed